webapp/src/ts/modals/session-expired/session-expired.component.ts

```diff
@@ -0,0 +1,24 @@
+import { MmModalAbstract } from '../mm-modal/mm-modal';
+import { SessionService } from '@mm-services/session.service';
+
+import { BsModalRef } from 'ngx-bootstrap/modal';
+import {Component} from '@angular/core';
+
+
+@Component({
+  selector: 'session-expired',
+  templateUrl: './session-expired.component.html'
+})
+export class SessionExpiredComponent extends MmModalAbstract {
+
+  constructor(
+    bsModalRef:BsModalRef,
+    private sessionService:SessionService,
+  ) {
+    super(bsModalRef);
+  }
+
+  submit() {
+    this.sessionService.navigateToLogin();
```

Comment:
I feel like we should close the modal here, like the old code did. What if we changed the `this.sessionService.navigateToLogin` to NOT reload the page, that would mean the modal would remain on the screen?